src/modules/formBuilder/components/itemEditor/FormEditorItemProperties.tsx

```diff
@@ -108,6 +136,15 @@ const FormEditorItemProperties: React.FC<FormEditorItemPropertiesProps> = ({
     [definition.definition]
   );
 
+  // If a field is marked as "always hidden", clear the enable_when conditions
```

Comment:
I could imagine this causing ux issues if someone taps the "always hidden" button accidentally (just clicking around) on a form item that already has a complicated enableWhen setup. I'm sure it's fine since they can just cancel their changes instead of saving. But maybe we could consider also adding helper text such as "Removes any existing conditions for visibility"? We even could set that text conditionally depending on `hasEnableWhen`.

Reply:
Hmm good point, I considered that too. I just reduced the click target on the "always hidden" checkbox so it only covers the label, not the full width, which might reduce accidental clicks. I will leave as-is for now since user can discard changes if anything was lost.